UITests/TransferFunds/TransferUserFundsTest.swift

```diff
@@ -472,7 +480,7 @@ class TransferUserFundsTest: BaseTests {
         let pastAmountWithNumberNoDigit = "10000"
         transferFunds.pasteAmountToTransferAmount(amount: pastAmountWithNumberNoDigit)
         transferFunds.transferAmount.tap()
-        XCTAssertEqual(transferFunds.transferAmount.value as? String, "10,000.00")
+        XCTAssertEqual(transferFunds.transferAmount.value as? String, "10000.00")
```

Comment:
10,000.00 is valid amount to be pasted.

Reply:
this assert is about what is in textField, not in the clipboard

Reply:
reverted